t/02-use.t

```diff
@@ -28,10 +28,12 @@ diag "no proj bin dir found via bin_dir method\n" if not @bin;
 my $info = eval {
     Alien::Proj4->load_projection_information
 };
-is $@, '', 'got projection information without error';
+my $e = $@;
+diag $e if $e;
+is $e, '', 'got projection information without error';
 #  could check some of the hash contents, but not sure it's worth it
 is ref $info, 'HASH', 'projection info is a hash ref';
-diag explain $info;
+diag explain [sort keys %$info];
```

Comment:
I'll be happy to accept this and push out a quick extra release once the above issue is sorted.

Reply:
Before I forget (I'm in the middle of wrenching PDLA::Core around to use Inline::Module and it's a big job): the above will either not work, or even blow up, if `$info` is anything other than a hash-ref, which is rather the problem we're trying to deal with.

Could you switch to something like this?

```
is ref $info, 'HASH', 'projection info is a hash ref'
  or diag explain $info;
```

I think that will give us more of what we need, while avoiding risk of blowing up!

Reply:
The main issue is that $info is massive when it works, and is empty on failure.  I'm not sure it will help with the diagnosis.  I'll close the PR.

Reply:
The above change would have made it diag only on failure, which is a good idea. I would be very pleased if you'd reopen, adjust as above, so I can release it? Else I'll just have to do it myself :-)

Reply:
OK, stand by for new/refreshed PR

Reply:
Embrace the force-push, Luke. Feel it flow through you! ;-)